.github/workflows/image-publish.yml

```diff
@@ -81,3 +96,18 @@ jobs:
         run: |
           SHORT_SHA1=$(git rev-parse --short=7 HEAD)
           docker buildx build --platform linux/amd64 -f build/dockerfiles/dev.Dockerfile --push -t quay.io/che-incubator/che-code-dev:insiders -t quay.io/che-incubator/che-code-dev:next -t quay.io/che-incubator/che-code-dev:insiders-${SHORT_SHA1} .
+      - name: Create failure MM message
+        if: ${{ failure() }} || {{ github.ref == 'refs/heads/main'}}
+        run: |
+          echo "{\"text\":\":no_entry_sign: Che Code ${{ github.event.inputs.version }} release has failed: https://github.com/che-incubator/che-code/actions/workflows/release.yml\"}" > mattermost.json
```

Comment:
`github.event.inputs.version` is not defined unless you add a `workflow_dispatch` with a `version` variable.